assets/src/domain/eventEditor/services/context/EntityListContext/withEntityListContext.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface ContextHOCArgs {
```

Comment:
maybe:

```suggestion
interface ContextHocArgs {
```

we have not discussed how we want to handle abbreviations within our naming schema
but we can discuss in chat since we currently don't have a set rule